### PR TITLE
Tickets/dm 31239

### PR DESCRIPTION
**Pull request by pcortes80**
Please check code. Thanks.

### CODE REVIEW

File: doc/versionHistory.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Version History
 
+0.2.1
+
+- Create sendEvt.vi to write the event to TCP Client.
```
Comment (teweitsai) [inline doc/versionHistory.md:5]: Please consider to use **sendEvt.vi** or `sendEvt.vi` to express the vi.

File: doc/versionHistory.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
...
+- Modify sendCmdStatus.vi to hide user event instance to the user.
```
Comment (teweitsai) [inline doc/versionHistory.md:6]: Please consider to use **sendCmdStatus.vi** or `sendCmdStatus.vi` to express the vi.

**Review by teweitsai — APPROVED** [review]
This update looks good to me. I left the inline comment for the document. For the code:

[ComponentSimulator.vi]

1. We should not need the "CmdWrite" from the getInstanceCmd.vi. Right? If this is the case, please remove it from getInstanceCmd.vi. In this case, the fuction document needs to be udpated as well.

2. We should not need the "EvtWrite" from the getInstanceEvt.vi. Right? If this is the case, please remove it from getInstanceEvt.vi. In this case, the fuction document needs to be udpated as well.

3. We should not need the "GenWrite" from the getInstanceGen.vi. Right? If this is the case, please remove it from getInstanceGen.vi. In this case, the fuction document needs to be udpated as well.

[sendEvt.vi]

1. I noticed that for the "sendEvt.vi" and "sendCmdStatus.vi", the front panel has only the "tcpServer out". I am not sure this is the intentional or not. But pleas have the "tcpServer in" and "tcpServer out" in the front panel as what we have in the "configServer.vi". This is to let the user have the flexibility to connect or use the tcpServer instance (or line).

[sendCmdStatus.vi]

1. As the comment in sendEvt.vi for the front panel: "tcpServer in" and "tcpServer out".

File: doc/versionHistory.md (hunk 1 above)
Comment (pcortes80) [reply doc/versionHistory.md:5]: Done.

File: doc/versionHistory.md (hunk 2 above)
Comment (pcortes80) [reply doc/versionHistory.md:6]: Done.

**Comment by pcortes80** [comment]
Here my replies:

[ComponentSimulator.vi]

We should not need the "CmdWrite" from the getInstanceCmd.vi. Right? If this is the case, please remove it from getInstanceCmd.vi. In this case, the fuction document needs to be udpated as well.

<Reply> I agree with you. I removed it. Updated documentation as well.

We should not need the "EvtWrite" from the getInstanceEvt.vi. Right? If this is the case, please remove it from getInstanceEvt.vi. In this case, the fuction document needs to be udpated as well.

<Reply> I agree with you. I removed it. Updated documentation as well.

We should not need the "GenWrite" from the getInstanceGen.vi. Right? If this is the case, please remove it from getInstanceGen.vi. In this case, the fuction document needs to be udpated as well.

<Reply> I agree with you. I removed it. Updated documentation as well.

[sendEvt.vi]

I noticed that for the "sendEvt.vi" and "sendCmdStatus.vi", the front panel has only the "tcpServer out". I am not sure this is the intentional or not. But please have the "tcpServer in" and "tcpServer out" in the front panel as what we have in the "configServer.vi". This is to let the user have the flexibility to connect or use the tcpServer instance (or line).

<Reply> It was a mistake. I fixed it.

[sendCmdStatus.vi]

As the comment in sendEvt.vi for the front panel: "tcpServer in" and "tcpServer out".

<Reply> It was a mistake. I fixed it.

**Pull request merged**